### DISCUSSION TITLE
Improve give command for enchanted items

### DISCUSSION THREAD

**Discussion by viasnake** [Zrips/CMI#7045] [Enhancement]
When giving an enchanted item to a player, the arguments for the give command differ slightly between essentials and CMI.
Therefore, some give commands written in the essentials style will not work properly with CMI.

I suggest that give commands written in the essentials style be modified to work with CMI as well.
Here is an example of the command

CMI
`/give [playerName] diamondpickaxe 1 e efficiency:1 e durability:1 e fortune:1`

essentials
`/give [playerName] diamondpickaxe 1 efficiency:1 durability:1 fortune:1`
When this command is executed at CMI, the player receives an unenchanted diamondpickaxe.

This change will allow for a seamless transition for some users migrating from essentials and allow existing cmi users to use shorter commands.

**Comment by mrfloris** [comment]
To be honest, personally, I feel that plugins that have these example commands in their configuration are fairly easy to customize and update to the syntax of other plugins. While it's convenient, it's also a one-time change. Just my personal opinion; I don't think this is really needed.

**Comment by MrCrazys** [comment]
> When giving an enchanted item to a player, the arguments for the give command differ slightly between essentials and CMI. Therefore, some give commands written in the essentials style will not work properly with CMI.
> 
> I suggest that give commands written in the essentials style be modified to work with CMI as well. Here is an example of the command
> 
> CMI `/give [playerName] diamondpickaxe 1 e efficiency:1 e durability:1 e fortune:1`
> 
> essentials `/give [playerName] diamondpickaxe 1 efficiency:1 durability:1 fortune:1` When this command is executed at CMI, the player receives an unenchanted diamondpickaxe.
> 
> This change will allow for a seamless transition for some users migrating from essentials and allow existing cmi users to use shorter commands.

Well, unfortunately that's just one of the things you have to get used to when migrating from plugins. I used to use /eco give in my configuration files for other plugins to give players money, but I had to instead use /cmi money give when I switched to CMI. This seems like an unncecessary change to me. You'll get used to it eventually.
